src/client/pages/Assessment/Assessment.tsx

```diff
@@ -69,6 +72,8 @@ const Assessment: React.FC = () => {
 
   if (!assessment) return null
 
+  if (!Authorizer.canView({ countryIso, assessment, cycle, user })) window.location.href = BasePaths.Root()
```

Comment:
`history.replace` ?

Reply:
it's complicated. we can'y use useHistory at the same level of <Routes> definition (it is null).

Reply:
useNavigate ? https://reactrouter.com/en/main/hooks/use-navigate

Reply:
just as info,
using `window.location.href` creates a new ajax query and leaves the SPA
while using the react-router stays inside SPA and does not create unnecessary ajax calls

Reply:
yes, that's why I'm suggesting that